src/js/utils/caching.js

```
@@ -8,7 +8,7 @@ export const cache = getConfiguredCache({
 });
 
 const actionsToPersist = {
-  // ...
+  PRODUCTS_TAB_ACTIVE: ['settings'],
 };
```

Wow, this is kinda magical (in a good way, I think)! How is browser support for IndexedDB?

---

Agreed! It's using https://github.com/HenrikJoreteg/redux-persist-middleware and https://github.com/HenrikJoreteg/money-clip...

Browser support is pretty good (https://caniuse.com/#feat=indexeddb), but there are polyfills if it becomes an issue. (It also fails silently if the cache doesn't work.)